serving/docker/lmi.Dockerfile

```diff
@@ -37,7 +37,7 @@ ARG flash_attn_2_wheel="https://github.com/vllm-project/flash-attention/releases
 ARG flash_infer_wheel="https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.6/flashinfer-0.1.6+cu124torch2.4-cp310-cp310-linux_x86_64.whl"
 # %2B is the url escape for the '+' character
 # TODO: update this back to nightly wheel once required deps have been updated
-ARG lmi_dist_wheel="https://publish.djl.ai/lmi_dist/lmi_dist-12.0.0-py3-none-any.whl"
+ARG lmi_dist_wheel="https://publish.djl.ai/lmi_dist/lmi_dist-11.0.0%2Bnightly-py3-none-any.whl"
```

Comment:
can we fix lmi-dist to start building 13.0 nightly?

Reply:
Updated to 13.0.0 nightly.